usr/src/cmd/install-tools/usbgen

```diff
@@ -374,7 +374,8 @@ if [ -f ${iso_path}/boot/bootia32.efi ] ; then
   cp ${iso_path}/boot/bootia32.efi ${usb_path}/efi/boot/BOOTIA32.EFI
   cp ${iso_path}/boot/bootx64.efi ${usb_path}/efi/boot/BOOTX64.EFI
 else
-  cp ${iso_path}/boot/boot1.efi ${usb_path}/efi/boot/BOOTX64.EFI
```

Comment:
Should we check if file exists and copy one or another depending on this to support older illumos-gate for some time?